gatorgrade/output/output_functions.py

```diff
@@ -1,2 +1,30 @@
-# This class is used for storing the main functions requested from the Github Issue Tracker for the output team.
-# For instance, functions dealing with percentage output, description output, and colorization of text.
\ No newline at end of file
+''' This module is used for storing the main functions requested from the Github Issue Tracker for the output team.
+For instance, functions dealing with percentage output, description output, and colorization of text. '''
+
+import gator
+import os
+import output_tools
+
+''' Commands are received as list of tuples.
+Each tuple in the list is a string and list of strings <string, [strings]>
+The string is the path to the file or filename and the list is the commands for the file. '''
+def receive_command(command_info : List[Tuple(str, List[str])]):

```

Comment:
please add a function docstring to this that provides a basic explanation of what the function is performing